🐛 BUG: [pages + windows] Could not automatically determine proxy port

### What version of `Wrangler` are you using?

2.0.24

### What operating system are you using?

Windows

### Describe the Bug

This specifically occurs when the directory which wrangler is being run in is in a different disk than the directory that netstat is located.

For example, in this case, netstat is located on my C: drive, whereas my code is located on my F: drive. It incorrectly searches the F drive for the file, which does not exist. Creating a symlink or copying the executable into the correct path does resolve this for me, however that's obviously not a real solution.

```
🚧 'wrangler pages <command>' is a beta command. Please report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose
Running npm run dev...
Sleeping 5 seconds to allow proxy process to start before attempting to automatically determine port...
To skip, specify the proxy port with --proxy.
[proxy]: 
> link-site@0.0.1 dev
> svelte-kit dev --host


No Instance(s) Available.
The system cannot find the path specified.
X [ERROR] Error scanning for ports of process with PID 5252: Error: Command failed: \windows\system32\netstat.exe -nao

  The system cannot find the path specified
```

--- a/.changeset/blue-ladybugs-attend.md
+++ b/.changeset/blue-ladybugs-attend.md
@@ -1,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+fix: [windows] unable to find netstat

--- a/packages/wrangler/src/pages/dev.tsx
+++ b/packages/wrangler/src/pages/dev.tsx
@@ -366,6 +366,14 @@ function getPort(pid: number) {
 
 	if (isWindows()) {
 		command = "\\windows\\system32\\netstat.exe -nao";
+		const path = process.env.PATH?.split(":");
+		if (path) {
+			// Aims to match `\\WINDOWS;C`, where C is the network drive letter.
+			const networkDrive = path.find((p) => /^\\WINDOWS;[A-Z]$/g.test(p));
+			if (networkDrive) {
+				command = networkDrive.charAt(networkDrive.length - 1) + ":" + command;
+			}
+		}
 		regExp = new RegExp(`TCP\\s+.*:(\\d+)\\s+.*:\\d+\\s+LISTENING\\s+${pid}`);
 	} else {
 		command = "lsof -nPi";

--- a/packages/wrangler/src/pages/dev.tsx
+++ b/packages/wrangler/src/pages/dev.tsx
@@ -1,6 +1,6 @@
 import { execSync, spawn } from "node:child_process";
 import { existsSync } from "node:fs";
-import { tmpdir } from "node:os";
+import { homedir, tmpdir } from "node:os";
 import { join, resolve } from "node:path";
 import { watch } from "chokidar";
 import { build as workerJsBuild } from "esbuild";
@@ -365,15 +365,8 @@ function getPort(pid: number) {
 	let command: string, regExp: RegExp;
 
 	if (isWindows()) {
-		command = "\\windows\\system32\\netstat.exe -nao";
-		const path = process.env.PATH?.split(":");
-		if (path) {
-			// Aims to match `\\WINDOWS;C`, where C is the network drive letter.
-			const networkDrive = path.find((p) => /^\\WINDOWS;[A-Z]$/g.test(p));
-			if (networkDrive) {
-				command = networkDrive.charAt(networkDrive.length - 1) + ":" + command;
-			}
-		}
+		const drive = homedir().split(":\\")[0];
+		command = drive + ":\\windows\\system32\\netstat.exe -nao";
 		regExp = new RegExp(`TCP\\s+.*:(\\d+)\\s+.*:\\d+\\s+LISTENING\\s+${pid}`);
 	} else {
 		command = "lsof -nPi";

